api/v1alpha1/gateway_types.go

```diff
@@ -113,8 +117,7 @@ const (
 	NamedAddress = "NamedAddress"
 )
 
-// ListenerAddress describes an address bound by the
-// Listener.
+// ListenerAddress describes an address bound by the Listener.
```

Comment:
Strangely this is actually both used to indicate bound and requested since it appears in spec and status. I've made this comment more generic since it can be used in either case.